core/client/app/services/clock.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+
+const {
+    Service,
+    run
+} = Ember;
+
+const ONE_SECOND = 1000;
+
+// Creates a clock service to run intervals.
+
+export default Service.extend({
+    second: null,
+    minute: null,
+    hour:   null,
+
+    init() {
+        this.tick();
+    },
+
+    tick() {
+        let now = moment.utc();
+        this.setProperties({
+            second: now.seconds(),
+            minute: now.minutes(),
+            hour:   now.hours()
+        });
+
+        let self = this;
```

Comment:
The downside to disabling the clock service during testing is that it can end up requiring a cascade of special casing for everything in the app that depends on it that you may want to test.

Another workaround for this issue is to run half the interval outside of Ember's run loop, so that you give it some cool down time and allow the wait helpers to progress.  Something like this is what I normally use:

```js
setTimeout(() => {
    if (this.isDestroying === false && this.isDestroyed === false) {
        run.later(() => {
            if (this.isDestroying === false && this.isDestroyed === false) {
                this.tick();
            }
        }, 500); // 500ms in the run loop
    }
}, 500); // 500ms of cool down to let things settle
```